src/docfx/lib/log/Error.cs

```diff
@@ -24,14 +24,17 @@ internal class Error
 
         public bool PullRequestOnly { get; }
 
+        public object[] OverrideParameters { get; }
```

Comment:
I didn't see any tests have `overrideParameters`, how does it works? I think we need code change once the parameters changed?